spec/index.bs

```diff
@@ -16778,7 +16782,18 @@ capabilities on below {{GPUTextureFormat}}s:
 - {{GPUTextureFormat/"r8snorm"}}
 - {{GPUTextureFormat/"rg8snorm"}}
 - {{GPUTextureFormat/"rgba8snorm"}}
-- {{GPUTextureFormat/"rg11b10ufloat"}}
+
+Enables {{GPUFeatureName/"rg11b10ufloat-renderable"}}, which means calling
+
+<pre highlight=js>
+    requestDevice({ requiredFeatures: ['texture-formats-tier1'] })
+</pre>
+
+gives you a device with
+
+<pre highlight=js>
+    device.features.has('rg11b10ufloat-renderable')
+</pre>
```

Comment:
nit: This example is a little informally written for spec text, and anyway I don't think it's really necessary. Just duplicating the same note from above would be clear and concise, I think.
```suggestion
Enabling {{GPUFeatureName/"texture-formats-tier1"}} at device creation will also enable
{{GPUFeatureName/"rg11b10ufloat-renderable"}}.
```